packages/klevu-core/src/connection/resultHelpers/getRedirects.ts

```diff
@@ -0,0 +1,16 @@
+import { KlevuKMCSettings } from "../kmc.js"
+import { KlevuKeywordUrlMap } from "../kmcmodels/KMCMaps.js"
+
+export async function getRedirects(
+  term: string
+): Promise<KlevuKeywordUrlMap[]> {
+  const settings = await KlevuKMCSettings()
+
+  if (!settings.maps) {
+    return []
+  }
+
+  return settings.maps.klevu_keywordUrlMap.filter((map) =>
+    map.keywords.some((k) => k === term)
```

Comment:
can map.keywords be null at any time?

Reply:
No. There is no option for term to be null.